src/esp/sensor/Sensor.cpp

```diff
@@ -12,7 +12,8 @@ namespace esp {
 namespace sensor {
 
 Sensor::Sensor(scene::SceneNode& node, SensorSpec::ptr spec)
-    : Magnum::SceneGraph::AbstractFeature3D{node}, spec_(std::move(spec)) {
+    : Magnum::SceneGraph::AbstractFeature3D{node},
+      spec_(std::move(spec)) {  // why are we moving?
```

Comment:
Performance. See clang-tidy performance checks.

Reply:
Moving a pointer makes that much of a difference?

Reply:
@jturner65 Feel free to remove it if you can make clang-tidy happy.

Reply:
@jturner65 I removed it.

Reply:
/home/circleci/project/habitat-sim/src/esp/sensor/Sensor.cpp:15:58: error: parameter 'spec' is passed by value and only copied once; consider moving it to avoid unnecessary copies [performance-unnecessary-value-param,-warnings-as-errors]

https://stackoverflow.com/questions/41871115/why-would-i-stdmove-an-stdshared-ptr
https://clang.llvm.org/extra/clang-tidy/checks/performance-unnecessary-value-param.html